### PR TITLE
feat(pointcloud): Support new calib - fisheye distortion

### PR DESCRIPTION
**Pull request by Kerwin-L**

### CODE REVIEW

File: packages/lb-annotation/src/core/pointCloud/matrix.ts [hunk 1]
```diff
@@ -64,6 +65,67 @@ export function rotatePoint(
...
+ */
+export const isFisheyeCalibValid = (calib: ICalib) => {
+  if (calib.fisheyeDistortion?.length > 0 && isMatrixValid(calib.P, 3, 4) && isMatrixValid(calib.T, 3, 4)) {
+    return true;
+  }
+
+  return false;
+};
+
+export const lidar2ImageByFisheye = (point: I3DSpaceCoord, calib: ICalib) => {
```
Comment (Glenfiddish) [inline packages/lb-annotation/src/core/pointCloud/matrix.ts:97]: How aboout renaming as lidarToFishEyeCood or lidar2FishEye2D? 
Add comment and returns.

Comment (Kerwin-L) [reply packages/lb-annotation/src/core/pointCloud/matrix.ts:97]: You are right. I think `lidar2FisheyeImage` is better.

**Pull request merged**